### PR TITLE
add rule for DiePresse.com

### PR DESCRIPTION
**Pull request by numismatika**

### CODE REVIEW

**Comment by zoracon** [comment]
Look into listing the hosts fully since the more complex regex requires many testing urls.

**Comment by zoracon** [comment]
Errors from last round:

```
ERROR src/chrome/content/rules/DiePresse.com.xml: Not enough tests (0 vs 8) for <Rule from '^http://(immobilien|karrierenews|schaufenster)\.diepresse\.com\.de/(css/|fragments/|images/|img/|js/|layout/)' to 'https://\g<1>.diepresse.com/\g<2>'>
ERROR src/chrome/content/rules/DiePresse.com.xml: Not enough tests (0 vs 2) for <Rule from '^http://diepresse\.com\.de/(assets/|layouts/)' to 'https://diepresse.com/\g<1>'>
```

File: src/chrome/content/rules/DiePresse.com.xml [hunk 1]
```diff
@@ -0,0 +1,106 @@
...
+<ruleset name="DiePresse.com (partial)">
+	<target host="diepresse.com"/>
+
+	<target host="abo.diepresse.com"/>
+	<target host="apps.diepresse.com"/>
+	<target host="dadat-boersespiel.diepresse.com"/>
+	<target host="immobilien.diepresse.com"/>
+	<target host="karrierenews.diepresse.com"/>
+	<target host="media.diepresse.com"/>
+	<target host="parship.diepresse.com"/>
```
Comment (ivysrono) [inline src/chrome/content/rules/DiePresse.com.xml]: https://travis-ci.org/EFForg/https-everywhere/jobs/586565459#L355

File: src/chrome/content/rules/DiePresse.com.xml [hunk 2]
```diff
@@ -0,0 +1,106 @@
...
+	<target host="schaufenster.diepresse.com"/>
+	<target host="service.diepresse.com"/>
+	<target host="shop.diepresse.com"/>
+	<target host="static.diepresse.com"/>
+
+	<securecookie host=".+" name=".+" />
+
+	<rule from="^http://(immobilien|karrierenews|schaufenster)\.diepresse\.com\.de/(css/|fragments/|images/|img/|js/|layout/)" to="https://$1.diepresse.com/$2"/>
```
Comment (ivysrono) [inline src/chrome/content/rules/DiePresse.com.xml]: rule from .com.de to .com, your tests are .com.

File: src/chrome/content/rules/DiePresse.com.xml [hunk 3]
```diff
@@ -0,0 +1,106 @@
...
+		<exclusion pattern="^http://(immobilien|karrierenews|schaufenster)\.diepresse\.com/$" />
+
+	<rule from="^http://diepresse\.com\.de/(assets/|layouts/)" to="https://diepresse.com/$"/>
```
Comment (ivysrono) [inline src/chrome/content/rules/DiePresse.com.xml]: `layouts` or `layout`?
to `$1`?

**Comment by pipboy96** [comment]
@zoracon All activity I see from this user is (possibly automated) force pushes. I'm not sure if the account is abandoned and it's ok to close all PRs by it. There were no response from them to any requests so I guess it's ok to wait a short time and close.

**Comment by ivysrono** [comment]
> 
> 
> @zoracon All activity I see from this user is (possibly automated) force pushes. I'm not sure if the account is abandoned and it's ok to close all PRs by it. There were no response from them to any requests so I guess it's ok to wait a short time and close.

agree. Anyone can pull new request if nessary.

**Comment by pipboy96** [comment]
@numismatika If you are going to continue work on your PRs simply send any comment.

**Pull request closed without merging**